Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInputEditor.cs

```diff
@@ -474,7 +474,7 @@ void AddEntry(InputAction action, PlayerInput.ActionEvent actionEvent)
             var selectedDefaultActionMap = !string.IsNullOrEmpty(playerInput.defaultActionMap)
                 ? asset.FindActionMap(playerInput.defaultActionMap)
                 : null;
-            m_SelectedDefaultActionMap = 0;
+            m_SelectedDefaultActionMap = asset.actionMaps.Count > 0 ? 1 : 0;
```

Comment:
Minor tweak to PlayerInputEditor so that if you select an .inputactions asset, it will default to the first action map in the asset (except if you already had an action map selected and there's one with the same name in the new asset).

Obsoletes the need to not forget to set the default action map. In practice, the majority of users will want an action map to be enabled by PlayerInput.